keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/StudentT.java

```diff
@@ -14,68 +14,65 @@
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 
 /**
- * Student T Distribution
- * https://en.wikipedia.org/wiki/Student%27s_t-distribution#Sampling_distribution
+ * @see "Computer Generation of Statistical Distributions
+ * by Richard Saucier
+ * ARL-TR-2168 March 2000
+ * 5.1.23 page 36"
  */
 public class StudentT implements ContinuousDistribution {
 
     private static final double HALF_LOG_PI = log(PI) / 2;
-    private final IntegerTensor v;
+    private final IntegerTensor alpha;
```

Comment:
As with my comment for the ChiSquared, I'd prefer `degreesOfFreedom`

Fun fact: it's not actually meant to be "v", it's the Greek letter "nu" which looks like a "v"